api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirEncounterDaoImpl.java

```diff
@@ -107,4 +111,9 @@ protected String paramToProp(@Nonnull String param) {
 				return null;
 		}
 	}
+	
+	@Override
+	protected Criterion generateNotCompletedOrderQuery(String path) {
+		throw new APIException("Not implemented until 2.2");
```

Comment:
Defer to @ibacher , but not sure throwing an Exception is the right thing here.  I though @ibacher  implied at one point that in FHIR it would just be silently ignored.

Reply:
Cool, yeah, happy to do whatever Ian thinks is correct.